src/Zttp.php

```diff
@@ -54,6 +54,11 @@ function asFormParams()
         return $this->bodyFormat('form_params')->contentType('application/x-www-form-urlencoded');
     }
 
+    function asMultipart()
+    {
+        return $this->bodyFormat('multipart')->contentType("multipart/form-data; boundary=".md5(uniqid(null, true)));
```

Comment:
This piece of code from Guzzle makes me think we can just not set the contentType stuff at all:

```php
        $request = Psr7\modify_request($request, $modify);
        if ($request->getBody() instanceof Psr7\MultipartStream) {
            // Use a multipart/form-data POST if a Content-Type is not set.
            $options['_conditional']['Content-Type'] = 'multipart/form-data; boundary='
                . $request->getBody()->getBoundary();
        }
```

Could you test just removing that entire function call and seeing if things work as you expect?